concourse/steps/release.py

```diff
@@ -717,27 +720,31 @@ def upload_ctf(ctf_path: str):
         else:
             upload_ctf(ctf_path=self.ctf_path)
 
-        try:
-            release = self.github_helper.repository.release_from_tag(release_tag_name)
+        def attach_cd_to_release():
```

Comment:
naming nits:
- rather not use `cd` as an abbreviation for component-descriptor
- how about: `upload_component_descriptor_as_release_asset`?